mattlianje/gregarious-wolf
# Fleshed out SQL connection architecture

1) Created sqlConnectionSetup function that checks if 'test_table' exists and creates it if it doesn't. Creates it with an auto-incremented primary key

2) Created urlExists function to check if URL is already in test_table.

**HOW TO TEST**

- Make sure you have imported pymysql and sqlalchemy
- Make sure you have your Apache and SQL turned on for XAMPP/MAMP/etc.
- On your localhost server, create a database called _climactic_test_
- Make sure to delete any table you have in _climactic_test_
- Make sure in `sqlConnection.py` you have your username and password setup correct for root@localhost
- `$ python main.py log`
- Choose option 1, import the following video: "https://www.youtube.com/watch?v=oK_rLUl2_8w"
- Now, re run the script and choose option 2. The csv is "highlights_data.csv"
- It should skip the first video because its already in the database
